### DISCUSSION TITLE
vscode c++ include path

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1402] [Q&A]
Hi, I want to develop the crazyflie firmware but why I can't read the head file. I added in the include path but still the same problem. plz to help. tkss

**Comment by gemenerik** [comment]
Can you elaborate a bit? Are you adding files? Did you include them in the Makefile?

**Reply by CalvinKoung** [reply]
For instance, I opened the mm_distance.c and I saw the top row have include mm_distance.h head file. But appear the red wave line below the code. 

Did you include them in the Makefile? (What is this means ? or how can I do ?)
thanks for help.

**Reply by knmcguire** [reply]
Hi! You might want to download the c/c++ extension of vscode which might be able to read that from the Makefile that is generated from kbuild. For the rest... vscode related issues are slightly out of the scope of these discussions.
